backend/src/main/java/gov/cdc/usds/simplereport/service/TestResultUploadService.java

```diff
@@ -87,7 +89,7 @@ public TestResultUpload processResultCSV(InputStream csvStream) {
         testResultFileValidator.validate(new ByteArrayInputStream(content));
     if (!errors.isEmpty()) {
       result.setErrors(errors.toArray(FeedbackMessage[]::new));
-      return result;
+      return new ResponseEntity<>(result, HttpStatus.OK); // should return bad request?
     }
```

Comment:
@zdeveloper do you have an opinion on this?

Reply:
(meaning should a validation failure be a 400 or should we just leave it as-is)

Reply:
I dont think the service should return a ResponseEntity, since its a controller concept. I think it should throw a custom exception e.g. _CsvProcessingException_ and notice how it sets the response code `@ResponseStatus(HttpStatus.BAD_REQUEST)`